cargo-dist/src/config/v1/mod.rs

```diff
@@ -481,6 +483,173 @@ impl TomlLayer {
             }
         }
     }
+
+    /// Determines whether the configured install paths are compatible with each other
+    pub fn validate_install_paths(&self) -> DistResult<()> {
+        if let Some(installers) = &self.installers {
+            if let Some(paths) = &installers.common.install_path {
+                if paths.len() > 1 && paths.contains(&InstallPathStrategy::CargoHome) {
+                    return Err(DistError::IncompatibleInstallPathConfiguration {});
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    // We say "value is being ignored", but it seems like we use the "ignored" values anyway?
+    // However, for the v0->v1 migration, I'm going for reproducing behavior
+    // over fixing bugs, so I'm keeping it this way for now. -@duckinator
+    fn merge_warn(name: &str, package_manifest_path: &Utf8Path) {
+        warn!("dist.{} is set, but this is only accepted in workspace.metadata (value is being ignored): {}", name, package_manifest_path);
+    }
+
+    /// Merge a workspace config into a package config (self)
+    pub fn merge_workspace_config(
+        &mut self,
+        workspace_config: &Self,
+        package_manifest_path: &Utf8Path,
+    ) {
+        // This is intentionally written awkwardly to make you update it
+        let Self {
+            config_version: _,
+            dist_version,
+            dist_url_override,
+            dist,
+            allow_dirty,
+            targets,
+            artifacts,
+            builds,
+            ci,
+            hosts,
+            installers,
+            publishers,
+        } = self;
+
+        // Check for global settings on local packages
+        if dist_version.is_some() {
+            Self::merge_warn("dist-version", package_manifest_path);
+        }
+        if dist_url_override.is_some() {
+            Self::merge_warn("dist-url-override", package_manifest_path);
+        }
+
+        // Arguably should be package-local for things like msi installers, but doesn't make sense for CI,
```

Comment:
can you say more about how this is helpful for msi installers? agree on maybe not changing this yet- honestly allow dirty is a messy feature haha so i'd rather see it phased out in favor of heavy handed "custom" features that can inline needed changes, etc

Reply:
This comment was copied verbatim from `v0.rs` and, in retrospect, I have no idea what it's trying to communicate.

Leaning towards just removing it.